torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -120,6 +121,11 @@ struct ConstantPropagator {
     for (size_t i = 0; i < outputs.size(); ++i) {
       auto new_output = tryInsertConstant(*graph, outputs[i]);
       if (new_output) {
+        GRAPH_UPDATE(
+            "Folding\n",
```

Comment:
@ZolotukhinM  @bzinodev Do we want to try keeping one transformation message on the same line vs splitting it across multiple lines (i.e. no `"Folding\n"`)?

Reply:
I don't have a strong opinion on this, so up to you :) Keeping it consistent across different passes would be nice though.

Reply:
I think removing /n is good idea